guides/common/modules/proc_restricting-customrepoid-to-a-specific-os-version-or-architecture-in-project.adoc

```diff
@@ -0,0 +1,23 @@
+[id="Restricting_a_{customrepoid}_to_a_Specific_OS_Version_or_Architecture_in_Project_{context}"]
+= Restricting A {customrepotitle} to a Specific Operating System or Architecture in {Project}
+
+You can configure {Project} to make a {customrepo} available only on hosts with a specific operating system version or architecture.
+For example, you can restrict a {customrepo} only to {RHEL}{nbsp}9 hosts.
```

Comment:
Please make this "client-content-dnf" only. See https://github.com/theforeman/foreman-documentation/blob/master/guides/common/attributes-base.adoc#L125

Reply:
Applied.